markets/spot-market/contracts/modules/WrapperModule.sol

```diff
@@ -67,7 +67,7 @@ contract WrapperModule is IWrapperModule {
         (amountToMint, fees, config) = MarketConfiguration.quoteWrap(
             marketId,
             wrapAmountD18,
-            Price.getCurrentPrice(marketId, Transaction.Type.WRAP)
+            Price.getCurrentPrice(marketId, Transaction.Type.WRAP, true)
```

Comment:
please, use a constant to give meaning to the `true` like 
`bool private constant _USE_STRICT_TOLERANCE = true;`